services/bit/bit-components.tester.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const Joi = require('joi')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('collection (valid)')
+  .get('/ramda/ramda.json')
+  .expectBadge({
+    label: 'components',
+    message: Joi.string().regex(/^[0-9]+$/),
```

Comment:
You'll want to use `isMetric` here from `test-validators.js`, especially since the service is using the `metric` formatter